src/recipes/modal.ts

```diff
@@ -0,0 +1,162 @@
+      boxShadow: 'none',
+      border: 'none',
+      display: 'block',
+      width: '100%',
+      padding: '8%',
+      boxSizing: 'border-box',
+      zIndex: 'popup',
+      backgroundColor: 'bg',
+      transition: 'all 0.4s ease-out',
+    },
+    title: {
+      //   fontWeight: 400, from
+      fontWeight: 700,
+      marginBottom: '40px',
+      textAlign: 'center',
+    },
+    text: {
+      marginBottom: '2em',
+    },
+    actions: {
+      textAlign: 'center',
+    },
+    content: {
+      maxWidth: '40em',
+      margin: '0 auto',
+      maxHeight: 'none',
+    },
+    close: {
+      position: 'absolute',
+      top: '-5px',
+      right: '-5px',
+      fontSize: '12px',
+      verticalAlign: 'middle',
+      textAlign: 'center',
+      padding: '10px',
+      display: 'none',
+      '& span': {
+        display: 'inline-block',
+        width: '11px',
+        height: '11px',
+        color: {
+          base: 'rgba(0, 0, 0, 0.3)',
+          _dark: 'rgba(255, 255, 255, 0.3)',
+        },
+        borderColor: {
+          base: 'rgba(0, 0, 0, 0.3)',
+          _dark: 'rgba(255, 255, 255, 0.3)',
+        },
+      },
+    },
+    closeX: {
+      /* extend click area */
+      padding: '10px 20px',
+      margin: '-10px -20px',
+      position: 'fixed',
+      top: 'calc(9px - 0.2em)',
+      right: '11px',
+      color: 'inherit',
+      textDecoration: 'none',
+    },
+  },
+  variants: {
+    id: {
+      welcome: {
+        root: { height: '100%', fontSize: '26px', paddingTop: '2em' },
+        title: { marginBottom: '0', lineHeight: 1 },
+        text: {
+          marginTop: '2em',
+          marginBottom: '2.5em',
+          fontSize: '75%',
+        },
+        actions: { fontSize: '0.75em' },
+      },
+      home: {
+        // unused, from .modal-home, not sure what this is for
+        root: {
+          marginLeft: '20px',
+          marginTop: '-5px',
+        },
+      },
+      auth: {},
+      customizeToolbar: {},
+      devices: {},
+      export: {},
+      feedback: {},
+      invites: {},
+      help: {},
+      settings: {},
+      signup: {},
+      testGestureDiagram: {},
+    },
+    stack: {
+      true: {
+        actions: {
+          textAlign: 'center',
+          '@media screen and (min-width: 480px)': {
+            display: 'flex',
+            justifyContent: 'center',
+            alignItems: 'center',
+            '& > *': {
+              marginTop: '1em',
+            },
+            '& > :not(:last-child)': {
+              marginRight: '1em',
+            },
+          },
+        },
+      },
+    },
+    center: {
+      true: {
+        root: {
+          textAlign: 'center',
+          left: '0',
+          right: '0',
+          margin: '0 auto',
+        },
+      },
+    },
+    opaque: {
+      true: {
+        root: {
+          animation: 'fadein 0.4s',
+        },
+      },
+    },
+  },
+  jsx: ['Devices', 'Help', 'ShortcutTable', 'Auth', 'ModalComponent'],
```

Comment:
What does `jsx` do?

Reply:
`jsx` doesn't really matter, I probably should have only put `ModalComponent` in it. It tells PandaCSS which components use the modal recipe, but it isn't necessary.
Here are the docs to it. https://panda-css.com/docs/concepts/recipes#advanced-jsx-tracking

Reply:
Okay, thanks. 

If we can omit this that would be great. Components can be removed or change names, so I think it's less than ideal to tightly couple the recipe to the components where it is used.